### PR TITLE
Namespace

### PR DESCRIPTION
**Pull request by jpc20**
Adds `/api/v1` before all of the routes. We will need to update our rails app once we merge this.
Also made some small refactors.

### CODE REVIEW

File: app/poros/nytimes_results.rb [hunk 1]
```diff
@@ -3,64 +3,49 @@
 
 class NytimesResults
   def get_obituaries
-    json = NytimesService.new.get_obituaries
-
-    json[:response][:docs].map do |obituary_data|
-
-      Obituary.new(format_obituary_data(obituary_data))
```
Comment (jpc20) [inline app/poros/nytimes_results.rb:10]: Refactored all of these to a helper method because it was repeated in all of the methods.

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'sinatra', require: 'sinatra/base'
+gem 'sinatra-contrib', require: 'sinatra/namespace'
```
Comment (jpc20) [inline Gemfile:4]: Needed to add this gem to be able to namespace in Sinatra

Comment (stellakunzang) [reply Gemfile:4]: oo cool

File: app/test/obituary_test.rb [hunk 3]
```diff
@@ -10,43 +10,42 @@ def app
 	end
 
   def test_it_returns_a_200
-  	get '/'
+  	get '/api/v1/'
```
Comment (stellakunzang) [inline app/test/obituary_test.rb:13]: Thank you! this is pretty repetitive and was probably a pain in the butt, but definitely takes our microservice to the next level. Will definitely be showing this off in the README

**Review by stellakunzang — APPROVED** [review]

File: Gemfile (hunk 2 above)
Comment (EricLarson2020) [reply Gemfile:4]: looking good

File: app/poros/nytimes_results.rb (hunk 1 above)
Comment (EricLarson2020) [reply app/poros/nytimes_results.rb:10]: Nice work Jack!

**Review by EricLarson2020 — APPROVED** [review]

**Pull request merged**